app/templates/components/supplementary-file-browser.hbs

```diff
@@ -5,9 +5,11 @@
 {{#if supplementList}}
     <h5>{{selectedFile.name}}</h5>
     <section>
-        <button class="show-more-button text-muted left" {{action 'moveLeft'}}>
-            <i class="fa fa-caret-left"></i>
-        </button>
+        {{#if showArrows}}
```

Comment:
Is there logic to remove only one direction of the buttons when there is no more to scroll?